packages/vscode-extension/src/chat/commands/nextstep/officeAddinSteps.ts

```diff
@@ -0,0 +1,201 @@
+    followUps: [
+      {
+        label: "@office /create",
+        command: "create",
+        prompt: "",
+      },
+    ],
+    condition: (status: WholeStatus) => !isProjectOpened(status),
+    priority: 0,
+  },
+  {
+    title: "Prerequisites",
+    description: (status: WholeStatus) =>
+      `Ensure the following requirements are met before you start building your Office Add-in. It seems you met the prerequisites error: ${
+        status.machineStatus.resultOfPrerequistes || ""
+      }. You can fix it and try again.`,
+    docLink:
+      "https://learn.microsoft.com/en-us/office/dev/add-ins/concepts/requirements-for-running-office-add-ins",
+    commands: [
+      {
+        title: "Check Prerequisites Again",
+        command: CHAT_EXECUTE_COMMAND_ID,
+        arguments: [CommandKey.ValidateGetStartedPrerequisites],
+      },
+    ],
+    followUps: [],
+    condition: (status: WholeStatus) =>
+      isProjectOpened(status) && !isPrequisitesCheckSucceeded(status),
+    priority: 0,
+  },
+  {
+    title: "Summary of README",
+    description: (status: WholeStatus) => {
+      // readme must exist because the condition has checked it
+      const readme = status.projectOpened!.readmeContent!;
+      let description = "";
+      let findFirstSharp = false;
+      for (const line of readme.split("\n")) {
+        if (line.trim().startsWith("#")) {
+          findFirstSharp = true;
+        }
+        if (!findFirstSharp) {
+          continue;
+        }
+        if (line.toLocaleLowerCase().includes("prerequisite")) {
+          break;
+        }
+        description += line.trim() + " ";
+      }
+      return description;
+    },
+    commands: [
+      {
+        title: "Open README",
+        command: CHAT_EXECUTE_COMMAND_ID,
+        arguments: [CommandKey.OpenReadMe],
+      },
+    ],
+    followUps: [],
+    condition: (status: WholeStatus) =>
+      isProjectOpened(status) &&
+      isPrequisitesCheckSucceeded(status) &&
+      isDidNoActionAfterScaffolded(status) &&
+      isHaveReadMe(status),
+    priority: 1,
+  },
+  {
+    title: "Install Dependencies",
+    description: `Install the dependencies for your Office Add-in project. It run ''npm install'' command to install all the dependencies in the terminal.`,
+    docLink: "",
+    commands: [
+      {
+        title: "Install Dependencies",
+        command: CHAT_EXECUTE_COMMAND_ID,
+        arguments: [CommandKey.installDependency],
+      },
+    ],
+    followUps: [],
+    condition: (status: WholeStatus) =>
+      isProjectOpened(status) &&
+      isPrequisitesCheckSucceeded(status) &&
+      !isDidNoActionAfterScaffolded(status) &&
+      !isDependenciesInstalled(status),
+    priority: 1,
+  },
+  {
+    title: "Preview in Local Environment",
+    description: `Preview in Local Environment makes debugging Office Add-in effortless. It behaves the same as you presss button F5 in Visual Studio Code. You can preview your Add-in in the desktop host application.`,
```

Comment:
Proposed:
Preview in Local Environment makes debugging Office Add-in effortless. It works like pressing F5 in Visual Studio Code and you can preview your Add-in in the desktop host application.